Library/Homebrew/hardware.rb

```diff
@@ -40,7 +44,14 @@ def cores
       end
 
       def bits
-        64
+        case RUBY_PLATFORM
+        when /\.x86_64/, /\.ppc64/ then 64
+        when /\.i\d86/, /\.ppc/ then 32
+        end
```

Comment:
I think this and the other `case` statement might need some tweaking, based on the following results for `RUBY_PLATFORM` (most from my local machine running 10.11.5):

- `universal.x86_64-darwin15` with system Ruby on 10.11.5
- `i686-darwin15.5.0` with `ruby187` from Homebrew
- `x86_64-darwin15.5.0` with `ruby193` and `ruby20` from Homebrew
- `x86_64-darwin15.0` with `ruby21` from Homebrew
- `x86_64-darwin15` with `ruby22` and `ruby` from Homebrew
- `x86_64-linux-gnu` with system Ruby on Debian 8

Most of these aren't matched by any of the regular expressions in the `case` statements.